templates/index.ftl

```diff
@@ -10,90 +10,63 @@
-                            <a href="${content.rootpath}learn/useCases/taskAssignmentOptimization.html">Task Assignment</a>,
-                            <a href="${content.rootpath}learn/useCases/schoolTimetabling.html">School Timetabling</a>,
-                            <a href="${content.rootpath}learn/useCases/cloudOptimization.html">Cloud Optimization</a>,
-                            <a href="${content.rootpath}learn/useCases/conferenceScheduling.html">Conference Scheduling</a>,
-                            Job Shop Scheduling, Bin Packing
-                            and <a href="${content.rootpath}learn/useCases/">many more</a>.
-                            Every organization faces such challenges:
-                            assign a limited set of <em>constrained</em> resources (employees, assets, time and/or money)
-                            to provide products or services.
-                            OptaPlanner delivers more efficient plans, which reduce costs and improve service quality.</p>
-                    </div>
-                    <div class="paragraph">
-                        <p>OptaPlanner is <strong>a lightweight, embeddable planning engine</strong>.
-                            It enables everyday <a href="${content.rootpath}compatibility/java.html">Java</a> programmers to solve optimization problems efficiently.
-                            It is also compatible with other JVM languages (such as <a href="${content.rootpath}compatibility/kotlin.html">Kotlin</a> and <a href="${content.rootpath}compatibility/scala.html">Scala</a>).
-                            Constraints apply on plain domain objects and can call existing code.
-                            There’s no need to input constraints as mathematical equations.
-                            Under the hood, OptaPlanner combines sophisticated Artificial Intelligence optimization algorithms
-                            (such as Tabu Search, Simulated Annealing, Late Acceptance and other metaheuristics)
-                            with very efficient score calculation and other state-of-the-art constraint solving techniques.</p>
-                    </div>
-                    <div class="paragraph">
-                        <p>OptaPlanner is <strong>open source software</strong>, released under <a href="${content.rootpath}code/license.html">the Apache License</a>.
-                            It is written in 100% pure Java™, runs on any JVM and is available in <a href="${content.rootpath}download/download.html">the Maven Central repository</a> too.
-                            It works with <a href="${content.rootpath}compatibility/quarkus.html">Quarkus</a> and <a href="${content.rootpath}compatibility/springBoot.html">Spring Boot</a>.</p>
-                    </div>
+            <div class="col-md-7">
+                <h1>OptaPlanner</h1>
+                <h2>AI constraint solver for plans and schedules</h2>
```

Comment:
Two lines for this looks ugly and takes too much space. Consider making this one sentence, possibly by leaving out "AI constraint solver". You can hide these ugly keywords in the text below.

Spring also doesn't say that they are an "application framework and inversion of control container".

Reply:
New proposal created, to be discussed at meeting.